server/odc-common/src/main/java/com/oceanbase/odc/common/unit/BinarySizeUnit.java

```diff
@@ -49,6 +51,15 @@ public enum BinarySizeUnit {
         }
     }
 
+    public static long convertTo(long originalSize, @NotNull BinarySizeUnit originalUnit,
```

Comment:
`BinarySizeUnit.GB.of(1).convert(BinarySizeUnit.B)`

which means 1GB to 1024 * 1024 * 1024 B

Reply:
OK，I will change